megamek/src/megamek/common/Infantry.java

```diff
@@ -414,12 +414,14 @@ public int getWalkMP(boolean gravity, boolean ignoreheat, boolean ignoremodulara
                 && (getMovementMode() != EntityMovementMode.INF_JUMP)) {
             mp = Math.max(mp - 1, 0);
         }
+        //  PL-MASC IntOps p.84
         if ((null != getCrew())
                 && hasAbility(OptionsConstants.MD_PL_MASC)
-                && ((getMovementMode() == EntityMovementMode.INF_LEG)
-                    || (getMovementMode() == EntityMovementMode.INF_JUMP))) {
+                && getMovementMode() == EntityMovementMode.INF_LEG
+                    && isConventionalInfantry()) {
```

Comment:
```suggestion
                && getMovementMode().isLegInfantry()
                && isConventionalInfantry()) {
```

No need for the extra spacing now.